examples/jsm/objects/BatchedMesh.js

```diff
@@ -611,25 +756,55 @@ class BatchedMesh extends Mesh {
 		const multiDrawStarts = this._multiDrawStarts;
 		const multiDrawCounts = this._multiDrawCounts;
 		const drawRanges = this._drawRanges;
+		const perObjectFrustumCulled = this.perObjectFrustumCulled;
+
+		// prepare the frustum
+		if ( perObjectFrustumCulled ) {
+
+			_projScreenMatrix
+				.multiplyMatrices( camera.projectionMatrix, camera.matrixWorldInverse )
+				.multiply( this.matrixWorld );
+			_frustum.setFromProjectionMatrix(
+				_projScreenMatrix,
+				_renderer.isWebGPURenderer ? WebGPUCoordinateSystem : WebGLCoordinateSystem
```

Comment:
I think something like `_renderer.backend !== undefined ? _renderer.backend.coordinateSystem : WebGLCoordinateSystem` would be more correct. /ping @gkjohnson

Reply:
I'm not familiar enough with WebGPURenderer to know

Reply:
@sunag What do you think?

Reply:
I think we can use _renderer.coordinateSystem, it is compatible with both renderers.